listenbrainz/webserver/views/settings.py

```diff
@@ -66,6 +66,16 @@ def set_troi_prefs():
     return jsonify(data)
 
 
+@settings_bp.route("/brainzplayer/", methods=["POST"])
+@login_required
+def get_brainzplayer_prefs():
+    current_brainzplayer_prefs = db_usersetting.get_brainzplayer_prefs(db_conn, current_user.id)
+    data = {
+        "brainzplayer": current_brainzplayer_prefs,
+    }
+    return jsonify(data)
+
```

Comment:
The response here is
```
{
  "brainzplayer": {
    "brainzplayer": {
      "soundcloudEnabled": false,
      "spotifyEnabled": true,
      "youtubeEnabled": true
    }
  }
}
```

Instead you can directly return current_brainzplayer_prefs
```suggestion
    return jsonify(current_brainzplayer_prefs)

```

Appropriate changes can be made in the frontend

Reply:
Also, this endpoint is not required since the required data is available in the context